tests/python/MypyBearTest.py

```diff
@@ -50,7 +52,8 @@ def test_py2_version(self):
                             ['print 123'])
 
     def test_bad_language(self):
```

Comment:
You can simply remove this test ; bad languages are not a bear problem now.

Reply:
Hm... actually this also test if `language` is not supported by the bear (required for 100% cov too)